pkg/hintrunner/zero/zerohint_others.go

```diff
@@ -3,9 +3,59 @@ package zero
 import (
 	"github.com/NethermindEth/cairo-vm-go/pkg/hintrunner/core"
 	"github.com/NethermindEth/cairo-vm-go/pkg/hintrunner/hinter"
+	"github.com/NethermindEth/cairo-vm-go/pkg/utils"
+	"github.com/NethermindEth/cairo-vm-go/pkg/vm/memory"
+
 	VM "github.com/NethermindEth/cairo-vm-go/pkg/vm"
+	f "github.com/consensys/gnark-crypto/ecc/stark-curve/fp"
 )
 
+func newMemcpyContinueCopyingHint(continueCopying hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "MemcpyContinueCopying",
+		Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
+			//> n -= 1
+			//> ids.continue_copying = 1 if n > 0 else 0
+
+			//> n-=1
+			n, err := ctx.ScopeManager.GetVariableValue("n")
+			if err != nil {
+				return err
+			}
+
+			felt := new(f.Element)
```

Comment:
could you rename it to something like ```newN``` or ```updatedN```?

Reply:
sure!

Reply:
done! ✅